src/output/distWriter.ts

```diff
@@ -76,7 +76,7 @@ export function distWriter(props: IOuputParserProps): DistWriter {
       if (userString.indexOf('$name') > -1) userString = userString.replace(/\$name/g, options.fileName);
       const absPath = ensureAbsolutePath(userString, root);
       const relativePath = ensureFuseBoxPath(path.relative(root, absPath));
-      const browserPath = options.publicPath ? options.publicPath + relativePath : relativePath;
+      const browserPath = options.publicPath ? path.join(options.publicPath, relativePath) : relativePath;
```

Comment:
It's on purpose with `+`
Now imagine windows users, they will get quite a surprise there in their index. On top of that, sometimes users would like to see `.` which path removes

Reply:
Then I'll update the code with more verbosity to add a forward slash if needed. I don't see why you would keep the `.` In browserpath tho